Zend/tests/stack_limit/stack_limit_010.phpt

```diff
@@ -20,7 +20,7 @@ $expectedMaxSize = match(php_uname('s')) {
         'amd64' => 512*1024*1024 - 4096,
         'i386' => 64*1024*1024 - 4096,
     },
-    'Linux' => match (getenv('GITHUB_ACTIONS')) {
```

Comment:
These worked on AppVeyor previously, which is why they were intentionally only skipped on GitHub Actions.